src/styles/main.scss

```diff
@@ -1,3 +1,3 @@
-body {
-  margin: 0;
-}
+@import "./utils/variables";
```

Comment:
```suggestion
@import "./utils/_variables";
```

your file in the units folder is named differently

Reply:
agree, leave it as it is